### DISCUSSION TITLE
How do I run Call of Duty Warzone?

### DISCUSSION THREAD

**Discussion by nnx0r** [Whisky-App/Whisky#466] [Q&A]
Hello,

I installed Whisky and Battle.net, followed the guide for Diablo IV to make the client run.
However, the game is unable to run:

<img width="401" alt="Screenshot 2023-10-01 at 22 09 51" src="https://github.com/Whisky-App/Whisky/assets/3460368/d1c6be8c-d269-4231-bbbb-dc9f9e59af86">

I'm linked to [here](https://account.battle.net/login/sso?token=EU-870fcfea2e6a0ae9b653f89b345215bc-300476989&ref=https%3A%2F%2Fnydus.battle.net%2FApp%2FenUS%2Fclient%2Ferror%2FBLZBNTBNA00000012%3FtargetRegion%3DEU%26code%3D18&blzcmp=app) by Blizzard, and I tried the repair tool and also uninstalling and installing the game again. Landed to the same error.

I am running:
- Whisky Version 2.0.1 (27)
- macOS 14.0 (23A344)
- M2 Max

Any help would be greatly appreciated.

**Comment by Mr-xiaoyang** [comment]
i think maybe your computer capabiliy is too bad(imean maybe)

**Reply by nnx0r** [reply]
How so, it's M2 Max with all of the highest specs possible for purchase.

**Reply by Mr-xiaoyang** [reply]
if i didnt understand wrong, you computer is a Mac,right? well,mac is  expensive indeed,but it doesnt mean it is good at run a  3a game  especially Call of duty,which is badly majorization. And im not sure if  Warzone can be run under the macOS, you can browse the internet to confirm M2's adventage and why it is so expensive

**Reply by Spl4tterz** [reply]
Have you attempted to install the game via steam, and using steam to begin the game session.

**Reply by nnx0r** [reply]
No, I haven't. Is it worth trying?

**Reply by jerrrrrrr483** [reply]
did you solved the problem？

**Comment by ShikiSuen** [comment]
Ricochet AntiCheat utilizes kernel-level driver.

Theoreotically, all main titles of Call of Duty since Vanguard are not wine-emulateable.

The last main title of Call of Duty which does not use Ricochet AntiCheat is Black Ops 5 (Cold War).
